docs/content/latest/admin/yb-master.md

```diff
@@ -25,45 +25,143 @@ $ ./bin/yb-master \
 --replication_factor=3 &
 ```
 
-## Help
+## Online help
 
-Use the **-\-help** option to see all the commands supported.
+Run `yb-master --help` to display the online help.
 
 ```sh
 $ ./bin/yb-master --help
 ```
 
-## Configuration flags
-
-Flag | Mandatory | Default | Description 
-----------------------|------|---------|------------------------
-`--master_addresses` | Y | N/A |Comma-separated list of all the RPC addresses for `yb-master` consensus-configuration. 
-`--fs_data_dirs` | Y | N/A | Comma-separated list of directories where the `yb-master` will place all it's `yb-data/master` data directory. 
-`--fs_wal_dirs`| N | Same value as `--fs_data_dirs` | The directory where the `yb-master` will place its write-ahead logs. May be the same as one of the directories listed in `--fs_data_dirs`, but not a sub-directory of a data directory. 
-`--log_dir`| N | Same value as `--fs_data_dirs`   | The directory to store `yb-master` log files.  
-`--max_clock_skew_usec` | N | 50000 (50ms) | The expected maximum clock skew between any two nodes in your deployment.
-`--rpc_bind_addresses`| Y |`0.0.0.0:7100` | Comma-separated list of addresses to bind to for RPC connections.
-`--server_broadcast_addresses`| N |`0.0.0.0:7100` | Public IP or DNS hostname of the server (along with an optional port).
-`--use_private_ip`| N |`never` | Determines when to use private IP addresses. Possible values are `never`,`zone`,`cloud` and `region`. Based on the values of the `placement_*` config flags listed in this table.
-`--webserver_interface`| N |`0.0.0.0` | Address to bind for server UI access.
-`--webserver_port`| N | `7000` | Monitoring web server port.
-`--webserver_doc_root`| N | The `www` directory in the YugabyteDB home directory | Monitoring web server home.
-`--replication_factor`| N |`3`  | Number of replicas to store for each tablet in the universe.
-`--placement_zone`| N |`rack1`  | Name of the availability zone or rack where this instance is deployed.
-`--placement_region`| N |`datacenter1`  | Name of the region or data center where this instance is deployed.
-`--placement_cloud`| N |`cloud1`  | Name of the cloud where this instance is deployed.
-`--logtostderr`| N | N/A  | Log to standard error.
-`--flagfile`| N | N/A  | Load flags from the specified file.
-`--version` | N | N/A | Show version and build info.
-`--yb_num_shards_per_tserver` | N | -1 | The number of shards per yb-tserver per table when a user table is created. Server automatically picks a valid default internally.
+## Syntax
+
+```sh
+yb-master [ option  ] | [ option ]
+```
+
+## Configuration options
+
+### General options
+
+#### --version
+
+Shows version and build info, then exits.
+
+#### --master_addresses
+
+Specifies a comma-separated list of all the RPC addresses for `yb-master` consensus-configuration. Mandatory.
```

Comment:
Move mandatory on a separate line?
Mandatory: yes
or 
Required: yes

Reply:
Good idea — I hadn't decided how to handle that.